modules/oe_content_event/oe_content_event.info.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - field_group:field_group
   - inline_entity_form:inline_entity_form
   - entity_reference_revisions:entity_reference_revisions
+  - oe_time_caching:oe_time_caching
```

Comment:
This module is not a strictly a dependency as using that plugin is optional. We should add a section in the readme about integration with link list and there we say that we need this module too, in case we wish to use it.